code/go/0chain.net/blobbercore/handler/storage_handler.go

```diff
@@ -90,8 +92,9 @@ func (fsh *StorageHandler) GetAllocationUpdateTicket(ctx context.Context, r *htt
 	if r.Method != "GET" {
 		return nil, common.NewError("invalid_method", "Invalid method used. Use GET instead")
 	}
-	allocationTx := r.FormValue("id")
-	allocationObj, err := fsh.verifyAllocation(ctx, allocationTx, false)
+	allocationId := r.FormValue("allocation_id")
+
+	allocationObj, err := fsh.verifyAllocation(ctx, allocationId, allocationId, false)
```

Comment:
why is it not?
```go
allocationObj, err := fsh.verifyAllocation(ctx, allocationId, allocationTx, false)
```

Reply:
I think this function is not used anywhere so code is incomplete here ( TODO is also there ). 

func (fsh *StorageHandler) GetAllocationUpdateTicket(ctx context.Context, r *http.Request) (interface{}, error) {
	if r.Method != "GET" {
		return nil, common.NewError("invalid_method", "Invalid method used. Use GET instead")
	}
	allocationId := r.FormValue("allocation_id")

	allocationObj, err := fsh.verifyAllocation(ctx, allocationId, allocationId, false)

	if err != nil {
		return nil, common.NewError("invalid_parameters", "Invalid allocation id passed."+err.Error())
	}

	// TODO

	return allocationObj, nil
}

Reply:
This function is getting allocationID only from user. 

func (fsh *StorageHandler) GetAllocationDetails(ctx context.Context, r *http.Request) (interface{}, error) {
	allocationId := r.FormValue("allocation_id")
	allocationObj, err := fsh.verifyAllocation(ctx, allocationId, allocationId, false)

	if err != nil {
		return nil, common.NewError("invalid_parameters", "Invalid allocation id passed."+err.Error())
	}

	return allocationObj, nil
}